lib/src/facts/windows/operating_system_resolver.cc

```diff
@@ -100,8 +100,8 @@ namespace facter { namespace facts { namespace windows {
         // Override default release with Windows release names
         auto version = result.release.substr(0, lastDot);
         bool consumerrel = (wmi::get(vals, wmi::producttype) == "1");
-        if (version == "6.4") {
```

Comment:
To me this code block looks like we never actually want to be returning the kernel version string for _any_ platform. It will do that as a default, but I think we might actually want to be catching as many as we can and returning a meaningful OS version.